src/MoveBall.js

```diff
@@ -0,0 +1,163 @@
+
+            props.afterGameRestart(false);
+        };
+    }, [props.shouldStop, props.shouldRestartGame]);
+
+    const getInitialValue = (ball) => {
+        const arrayOfObjects = [];
+
+        for (let i = 0; i < props.colSize; i++) {
+            if (i === INITIAL_VALUE) {
+                arrayOfObjects.push(ball);
+            } else {
+                arrayOfObjects.push({ isPlayerOneBall: false, isPlayerTwoBall: false });
+            };
+        };
+
+        return arrayOfObjects;
+    };
+
+    const [array, setArray] = useState(getInitialValue(playerOneBall));
+
+    const handlePressKey = (event) => {
+
+        const currentPlayer = ballPlayer.isPlayerOneBall ? 'playerOne' : 'playerTwo';
+        const currentKeys = playerMovement[currentPlayer];
+        const { rightMove, leftMove, downMove } = currentKeys;
+
+        if (!props.shouldStop) {
+            switch (event.keyCode) {
+                case rightMove:
+                    moveRight(event);
+                    break;
+                case leftMove:
+                    moveLeft(event);
+                    break;
+                case downMove:
+                    moveDown(event);
+                    break;
+                default:
+                    break;
+            };
+        };
+    };
+
+    const getCurrentBallPosition = () => {
+        return array.findIndex((object) => (
+            object.isPlayerOneBall || object.isPlayerTwoBall
+        ));
+    };
+
+    const moveRight = () => {
+        const copy = [...array];
+        const currentBallPosition = getCurrentBallPosition();
+
+        if (currentBallPosition !== copy.length - 1) {
+            copy[currentBallPosition + 1] = { ...copy[currentBallPosition] };
+            copy[currentBallPosition] = { isPlayerOneBall: false, isPlayerTwoBall: false };
+        };
+
+        setArray(copy);
+    };
+
+    const moveLeft = () => {
+        const copy = [...array];
+        const currentBallPosition = getCurrentBallPosition();
+
+        if (currentBallPosition !== 0) {
+            copy[currentBallPosition - 1] = { ...copy[currentBallPosition] };
+            copy[currentBallPosition] = { isPlayerOneBall: false, isPlayerTwoBall: false };
+        };
+
+        setArray(copy);
+    };
+
+    const moveDown = () => {
+
+        const position = getCurrentBallPosition();
+        const activeElement = array[position];
+
+        if (!props.handleIsLastCellFull(position)) {
+            props.handleMoveDown(position, activeElement);
+
+            if (ballPlayer.isPlayerOneBall) {
+                setBallPlayer(playerTwoBall);
+                setArray(getInitialValue(playerTwoBall));
+            } else {
+                setBallPlayer(playerOneBall);
+                setArray(getInitialValue(playerOneBall));
+            };
+        };
+    };
+
+    return (
+        <span className={'firstLine-container'} onKeyDown={(e) => { handlePressKey(e) }} tabIndex={0} ref={ref} >
```

Comment:
This should be a div. The div tag is used in HTML to make divisions of content on the page